lib/provider/camunda/CamundaPropertiesProvider.js

```diff
@@ -448,9 +471,16 @@ function createExtensionElementsGroups(element, bpmnFactory, elementRegistry, tr
  * @param {Translate} translate
  */
 function CamundaPropertiesProvider(
-    eventBus, canvas, bpmnFactory,
-    elementRegistry, elementTemplates, translate) {
-
+    eventBus,
+    canvas,
+    bpmnFactory,
+    elementRegistry,
+    elementTemplates,
+    modeling,
+    replace,
+    selection,
+    translate
```

Comment:
Should we indicate this as breaking change? Because updates are needed when this provider is used.

Reply:
Yeah, we should probably do this. To avoid breaking your custom provider that extends this one you can do:

```javascript
super(...CamundaPropertiesProvider.$inject.map(dependency => injector.get(dependency)));
```

Reply:
I've added a note about the breaking change to the commit.